content/en/ecosystem/adopters.md

```diff
@@ -21,9 +21,9 @@ To have your organization listed, submit a PR with an entry added to the
 The entry must include the following[^grace-period-2024-01-01]:
 
 - Link to a blog post that describes how your organization makes use of
-  OpenTelemetry
+  OpenTelemetry.
 - GitHub handle or email address as a point of contact so that we can reach out
-  in case we have questions
+  in case we have questions.
```

Comment:
@chalin @cartermp my feeling for the English language is not good enough here: are items of an itemized list of non-sentences terminated by a dot? If so, we have to change this on a few other pages as well.

Reply:
Usually, one doesn't end with punctuation any list items that are incomplete sentences. Then again, you might want to do it for consistency if items are mixed. The Splunk Style Guide says:

> If the list items are complete sentences, include end punctuation.
> If the list items are characters, words, or phrases, don't include end punctuation.
> Use parallel sentence construction in list items. For example, start all list items with verbs, or end all list items with punctuation.

Reply:
Ok, this sounds like a reasonable norm to me (since I would do it that way as well in German) 👍

Reply:
Thanks for the reminder, I also agree with the suggested guidelines above.